agentforce/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,144 @@
+= Agentforce Connector 1.0 - Mule 4
+:lc-connector-name: agentforce
+
+Anypoint Connector for Agentforce (Agentforce Connector) provides connectivity to the https://help.salesforce.com/s/articleView?id=sf.generative_ai_trust_layer.htm&type=5[Salesforce Einstein Trust Layer]. 
```

Comment:
This definition will need to be expanded as the connector will not only allow for Anypoint developers to integrate their applications with LLMs via the Einstein Trust Layer (original Einstein AI Connector functionality) but also invoke agents running in Salesforce.